### PR TITLE
Feature/gp2 857 site footer

### PR DESCRIPTION
**Pull request by jakehopking**
## To do (delete all that do not apply):

 - [X] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-857
 - [X] Jira ticket has the correct status.
 - [X] [Changelog](CHANGELOG.md) entry added.
 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templates/core/footer.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </li>
+      <li>
+        <a href="{% url_map 'terms' %}">Terms and conditions</a>
+      </li>
+      <li>
+        <a href="#">Contact us</a>
+      </li>
+    </ul>
+    <div class="magna-footer__great-logo">
+      <a href="/">
```
[the remark this answers is not in the document]
Comment (jakehopking) [reply core/templates/core/footer.html]: Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+
```
Comment (depsiatwal) [inline CHANGELOG.md:2]: Can we add a CL entry please

File: core/templates/core/footer.html [hunk 3]
```diff
@@ -5,17 +5,17 @@
   <nav class="magna-footer__nav bg-blue-deep-90 text-white">
     <ul class="magna-footer__links footer-links">
       <li>
-        <a href="{% url_map "privacy" %}">Privacy notice</a>
+        <a href="#">Privacy notice</a>
       </li>
       <li>
-        <a href="{% url_map 'terms' %}">Terms and conditions</a>
```
Comment (bobmeredith) [inline core/templates/core/footer.html]: This should not cause a breakage with original code - but the value of the url 'terms' should be populated into constants and put in the lookup map in the url_map template tag.

File: core/templates/core/footer.html [hunk 4]
```diff
@@ -5,17 +5,17 @@
   <nav class="magna-footer__nav bg-blue-deep-90 text-white">
     <ul class="magna-footer__links footer-links">
       <li>
-        <a href="{% url_map "privacy" %}">Privacy notice</a>
```
Comment (bobmeredith) [inline core/templates/core/footer.html]: This should not cause a breakage with original code - but the value of the url 'privacy' should be populated into constants and put in the lookup map in the url_map template tag.

File: core/templates/core/footer.html [hunk 5]
```diff
@@ -5,17 +5,17 @@
...
       <li>
-        <a href="{% url_map 'terms' %}">Terms and conditions</a>
+        <a href="#">Terms and conditions</a>
       </li>
       <li>
         <a href="#">Contact us</a>
       </li>
     </ul>
     <div class="magna-footer__great-logo">
-      <a href="{% url home %}">
```
Comment (bobmeredith) [inline core/templates/core/footer.html]: This is a link to the homepage. It's a wagtail page in fact so a reverse lookup won't work.  Suggest you just use '/' as in the header.

**Review by bobmeredith — CHANGES_REQUESTED** [review]

File: core/templates/core/footer.html (hunk 5 above)
Comment (stevejalim) [reply core/templates/core/footer.html]: Yeah, even though django-doctor said not to use "/" I think the advice the bot gave wasn't useful here. "/" is a VERY common (and stable) approach to linking to the homepage

File: CHANGELOG.md (hunk 2 above)
Comment (jakehopking) [reply CHANGELOG.md:2]: What does this mean?

File: core/templates/core/footer.html (hunk 5 above)
Comment (jakehopking) [reply core/templates/core/footer.html]: Exactly @stevejalim - I had it set as "/"  before this bot suggested something that caused (indirectly) build errors.

File: core/templates/core/footer.html (hunk 4 above)
Comment (jakehopking) [reply core/templates/core/footer.html]: Sorry @bobmeredith I don't understand your comment regarding constants here. I was under the impression that @depsiatwal was going to do this kind of work as part of the BE task associated with this.

Comment (jakehopking) [reply core/templates/core/footer.html]: Updated to single quotes.

Comment (jakehopking) [reply core/templates/core/footer.html]: @bobmeredith, as mentioned @depsiatwal has added these as constants already: https://uktrade.atlassian.net/browse/GP2-936

File: core/templates/core/footer.html (hunk 3 above)
Comment (jakehopking) [reply core/templates/core/footer.html]: https://uktrade.atlassian.net/browse/GP2-936

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**